canvasapi/requester.py

```diff
@@ -159,7 +167,16 @@ def request(
         :type json: `bool`
         :rtype: :class:`requests.Response`
         """
-        full_url = _url if _url else "{}{}".format(self.base_url, endpoint)
+        # Check for specific URL endpoints available from Canvas. If not
+        # specified, pass the given URL and move on.
+        if _url == "new_quizzes":
+            request_url = "{}{}".format(self.new_quizzes_url, endpoint)
+        elif _url == "graphql":
+            request_url = self.graphql
+        else:
+            request_url = _url
+
+        full_url = request_url if _url else "{}{}".format(self.base_url, endpoint)
```

Comment:
Since we're doing multiple checks now, the inline if/else is a bit harder to read. How about something like this?

```suggestion
        if not _url:
            full_url = "{}{}".format(self.base_url, endpoint)
        elif _url == "new_quizzes":
            full_url = "{}{}".format(self.new_quizzes_url, endpoint)
        elif _url == "graphql":
            full_url = self.graphql
        else:
            full_url = _url
```

Reply:
Yep...makes more sense to check if the argument is even included first.